wavs-staking-template/cli.toml

```diff
@@ -0,0 +1,76 @@
+# these values can be overriden by environment variables
+# by following the pattern WAVS_CLI_<UPPERCASE_KEY>
+# so for example, the key `http_endpoint` can be overriden by setting the environment variable `WAVS_CLI_WAVS_ENDPOINT`
+# for arrays use a comma-separated list in a single string
+
+# The log level, in the format of tracing directives. Default is ["info"]
+# See https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html#directives for more information
+# e.g. WAVS_CLI_LOG_LEVEL="info, wavs=debug"
+log_level = ["info", "wavs=debug"]
+
+# the directory to store the data. Default is "/var/wavs-cli"
+# data = "~/wavs/cli"
+
+# wavs endpoint
+# wavs_endpoint = "http://localhost:8000"
+
+[chains.cosmos.layer-local]
+chain_id = "slay3r-local"
+bech32_prefix = "layer"
+rpc_endpoint = "http://localhost:26657"
+grpc_endpoint = "http://localhost:9090"
+gas_price = 0.025
+gas_denom = "uslay"
+faucet_endpoint = "http://localhost:8000"
+
+# localnode is local testing inside localnode docker-compose network
+[chains.cosmos.layer-localnode]
+chain_id = "layer-local"
+bech32_prefix = "layer"
+rpc_endpoint = "http://layer:26657"
+grpc_endpoint = "http://layer:9090"
+gas_price = 0.025
+gas_denom = "ulayer"
+faucet_endpoint = "http://faucet:8000"
+
+# hacknet is our public facing pseudo-testnet for hackathons
+[chains.cosmos.layer-hacknet]
+chain_id = "layer-hack-1"
+bech32_prefix = "layer"
+rpc_endpoint = "https://rpc.hack.layer.xyz:443"
+grpc_endpoint = "https://grpc.hack.layer.xyz:443"
+gas_price = 0.025
+gas_denom = "ulayer"
+faucet_endpoint = "https://rabbit.hack.layer.xyz"
+
+[chains.cosmos.neutron]
+chain_id = "pion-1"
+bech32_prefix = "neutron"
+rpc_endpoint = "https://rpc-falcron.pion-1.ntrn.tech"
+grpc_endpoint = "http://grpc-falcron.pion-1.ntrn.tech:80"
+gas_price = 0.0053
+gas_denom = "untrn"
+
+[chains.eth.local]
+chain_id = "31337"
+ws_endpoint = "ws://localhost:8545"
+http_endpoint = "http://localhost:8545"
+aggregator_endpoint = "http://localhost:8001"
```

Comment:
aggregator is now set on a per-service rather than per-chain basis (it's in the Service json)